src/components/FinalPage/Confirmation.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./Confirmation.css";
+import Header from "../Header/Header";
+import PetsIcon from "@mui/icons-material/Pets";
+export default function Confirmation() {
+  return (
+    <div style={{height:"100vh",width:"20vw",display:"flex",alignItems:"center",margin:"auto"}}>
+      <div className="Header-Div" >
```

Comment:
class names case is not right!!
```suggestion
      <div className="header-div" >
```

Reply:
@ZainShahbaz btw why aren't you using material ui?
I you were it should be like this
```suggestion
      <div className={classes.headerDiv} >
```